metadata-ws/src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/Sample.java

```diff
@@ -54,12 +56,15 @@
     @JsonProperty
     private String name;
 
-    Sample() {
-    }
+    @ApiModelProperty(position = 4, dataType = "java.lang.String", example = "[url1, url2]", notes = "Url to Taxonomy List")
+    @JsonProperty
+    @ManyToMany
+    @NotNull
+    @Column(nullable = false)
+    @Size(min = 1)
+    private List<Taxonomy> taxonomies;
 
-    public Sample(AccessionVersionId accessionVersionId, String name) {
```

Comment:
Why deleting this constructor?

Reply:
It gives a loophole to create sample without taxonomy and we have not used constructor anywhere